packages/frontend/app/(dashboard)/organization/course/add/page.tsx

```diff
@@ -86,7 +86,13 @@ export default function AddCoursePage(): ReactElement {
     API.semesters
       .get(userInfo.organization?.orgId || -1)
       .then((semesters) => {
-        setOrganizationSemesters(semesters)
+        setOrganizationSemesters(
+          semesters.filter(
+            (semester) =>
+              new Date(semester.endDate) > new Date(1970) &&
```

Comment:
```suggestion
              new Date(semester.endDate) > new Date(1971) &&
```

I'm assuming that new Date(1970) is Jan 1970, and the Test Course semester has an end date in July. So it will probably need to be 1971 otherwise the Test Course won't show up as an option
<img width="1211" height="439" alt="image" src="https://github.com/user-attachments/assets/f2202168-6ae8-4b25-82f3-8aa0c1f11e06" />